res_partner_fiscal_position/model/res_partner.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    OpenERP, Open Source Management Solution
+#    This module copyright (C) 2015 Therp BV (<http://therp.nl>).
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+from openerp import models, api
+
+
+class ResPartner(models.Model):
+    _inherit = 'res.partner'
+
+    """
+    when changing the partner's country update the 
+    property_account_position to the one of that country
+    """
+    @api.onchange('country_id')
+    def on_change_country_id(self):
```

Comment:
Now that there can be multiple onchange methods for the same field independent of each other, it is time to make the method names more specific. Chances are that some day, this module gets installed with another module that also implements an onchange method on the partner's country_id. Therefore, to prevent any name clashes, rename the method to something less generic (e.g. get_country_fiscal_position).